### PR TITLE
Enable checks for the lab 1

### PR DESCRIPTION
**Pull request by NovIrina**

### CODE REVIEW

File: config/stage_1_style_tests/check_lint.py [hunk 1]
```diff
@@ -98,15 +115,26 @@ def main() -> None:
...
-                        ],
-                        pyproject_path)
+            if repository_type == "public":
+                completed_process = check_lint_on_paths(
+                            [
+                                lab_path
+                            ],
+                            pyproject_path, ignore_tests=True)
+            else:
+                completed_process = check_lint_on_paths(
```
Comment (demid5111) [inline config/stage_1_style_tests/check_lint.py:133]: Remove if: run(..., ignore_tests=name==public)

File: config/stage_1_style_tests/check_lint.py [hunk 2]
```diff
@@ -34,6 +37,8 @@ def check_lint_on_paths(paths: list[Path], path_to_config: Path,
         "--rcfile",
         str(path_to_config)
     ]
+    if ignore_tests:
+        lint_args.extend(["--ignore", "tests"])
```
Comment (demid5111) [inline config/stage_1_style_tests/check_lint.py:41]: Why without lab name?

**Review by demid5111 — CHANGES_REQUESTED** [review]

File: config/stage_1_style_tests/check_lint.py (hunk 2 above)
Comment (NovIrina) [reply config/stage_1_style_tests/check_lint.py:41]: According to pylint documentation it is a basename, not path

File: config/stage_1_style_tests/check_lint.py (hunk 1 above)
Comment (NovIrina) [reply config/stage_1_style_tests/check_lint.py]: done

**Pull request merged**